solutions/fully-configurable/variables.tf

```diff
@@ -27,10 +27,15 @@ variable "existing_secrets_manager_crn" {
 
 variable "prefix" {
   type        = string
-  description = "The prefix to be added to all resources created by this solution. To skip using a prefix, set this value to null or an empty string. The prefix must begin with a lowercase letter and may contain only lowercase letters, digits, and hyphens '-'. It should not exceed 16 characters, must not end with a hyphen('-'), and can not contain consecutive hyphens ('--'). Example: prod-us-south."
   nullable    = true
+  description = "The prefix to be added to all resources created by this solution. To skip using a prefix, set this value to null or an empty string. The prefix must begin with a lowercase letter and may contain only lowercase letters, digits, and hyphens '-'. It should not exceed 16 characters, must not end with a hyphen('-'), and can not contain consecutive hyphens ('--'). Example: test-02-sme. [Learn more](https://terraform-ibm-modules.github.io/documentation/#/prefix.md)."
```

Comment:
Please rename the example to - `test-02-sm-engine` or similar one else this will fail precommit based on latest changes.
[Reference](https://ibm-cloudplatform.slack.com/archives/CG06FQ31P/p1755714387489339?thread_ts=1755595422.341259&cid=CG06FQ31P)